frontend/packages/dev-console/integration-tests/features/topology/topology-smoke_test.feature

```diff
@@ -0,0 +1,165 @@
+   And user can see traffic distribution from knative sevice app to its revisions with it's percentage number
+   And pod ring associated with revisions are present around node with color according to the pod status
+   And knative revision can have application url on top-right of the node 
+   And user sees edit source code decorator is on bottom right of the revision which can lead to github or che workspace
+   And user sees build decorator on bottom left on knative service app which will take user to either build tab 
+   And user checks revisions having label "REV" and then the name
+   Then user checks knative service having label "KSVC" and then the name of service
+
+@regression, @smoke
+Scenario: Sidebar of worload
+   Given user is in the topology with workloads
+   When user clicks on workload
+   And right sidebar opens with Resources tab selected by default 
+   And user checks for sidebar tabs as Details, Resources and Monitoring
+   And user verifies name of the node and Action menu present on top of the sidebar
+   And user may sees health check notifiation above the tabs for deployment and deployment-config
+   Then user checks for close button on top right corner of sidebar 
+   
+@regression, @smoke
+Scenario: Create visual connection between two nodes
+   Given user is in the topology with three nodes
+   When user opens sidebar of one of the node
+   And user opens action menu and selects "Edit Annotations" option
+   And user enters key vlue as "app.openshift.io/connects-to" and value as name of the node to which it will be associated
+   And user can see the arrow connecting them with head pointing to the node whose value is provided in the "Edit Annotations"
+   And user click on the front of arrow and drag it on to the third node and drop it
+   Then user can see the arrow connecting them with head pointing to the node where the arrow is dropped
+
+@regression, @smoke
+Scenario: context menu of nodes
+   Given user is in the topology
+   When user right clicks on the node 
+   And user sees context menu
+   Then user checks options
+
+@regression, @smoke
+Scenario: Views in topology
+   Given user is in topology with workloads 
+   When user clicks on Zoom In option
+   And user sees the chart area is zoomed
+   And user clicks on Zoom Out option
+   And user sees the chart area is zoomed out
+   And user clicks on Fit to Screen option
+   And user sees the nodes fitting within chart area
+   And user clicks on Reset View option
+   Then user sees the chart area is reset to original
+
+@regression, @smoke
+Scenario: Verify Application grouping
+   Given User is in topology with an application cluster
+   When User clicks on an applicaton grouping
+   And User can see application sidebar
+   And user can confirm the workload information present under resources in the sidebar
+   And user can view "Add to Application" and "Delete Application" in the Action menu
```

Comment:
```suggestion
   And user can see "Add to Application" and "Delete Application" in the Action menu
```

Reply:
What difference would it create?

Reply:
Not difference but to view something, we have to open it and if it is already there, we see it without performing any action. I felt that it should be replaced with see but if you feel whatever I have shared is right then you can update it otherwise I am okay with it too.